src/components/AboutUs/index.tsx

```diff
@@ -1,58 +1,63 @@
 import React from 'react';
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 import Button from '../Button'
 import { mediaQueries } from '../../mediaqueries';
 
+export interface InfoSectionProps {
+  primary?: boolean,
+  logo?: any;
+  header?: string;
+  subheader?: string;
+  content?: string[];
+  footerLogo?: any;
+  cta?: boolean,
+  ctaText?: string
+}
 
-const Container = styled.div`
+const Container = styled.div<InfoSectionProps>`
 display: flex;
 flex-direction: column;
-padding:2rem;
 letter-spacing: 2px;
 line-height: 1.8em;
 align-items: center;
+padding:2rem;
+background: ${props => props.primary ? "red" : "yellow"}
 `
 const LogoImage = styled.img`
 width: 25%;
 padding: 2rem;
-border-radius:60%;
+border-radius: 60%;
 `
-const Content = styled.div`
+const Content = styled.div<InfoSectionProps>`
```

Comment:
dont pass any props if you dont need them
```suggestion
const Content = styled.div`
```